ci-operator/step-registry/image-based-install-operator/reinstall/create-secrets/image-based-install-operator-reinstall-create-secrets-commands.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install create identity secrets command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOF"
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }') 2>&1
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
```

Comment:
Consider adding a comment saying what is going on here and why  this is a good way to test reinstall

Reply:
Added some more info to the step description